src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -58,9 +58,10 @@ paths:
 
   /person:
     $ref: 'paths/person.yaml'
-
   /person/{login}/token:
     $ref: 'paths/person_login_token.yaml'
+  /person/{login}/group:
+    $ref: 'paths/person_login_group.yaml'
```

```suggestion
  /person/{login}/group:
    $ref: 'paths/person_login_group.yaml'
  /person/{login}/token:
    $ref: 'paths/person_login_token.yaml'
```
Nitpick: I would order the endpoints alphabetically.

---

good practice, done